src/NexusMods.Settings/SettingsBuilder.cs

```diff
@@ -23,8 +25,45 @@ internal class SettingsBuilder : ISettingsBuilder
         return this;
     }
 
+    public ISettingsBuilder ConfigureStorageBackend<TSettings>(
+        Action<ISettingsStorageBackendBuilder<TSettings>> configureStorageBackend)
+        where TSettings : class, ISettings, new()
+    {
+        var builder = new SettingsStorageBackendBuilder<TSettings>();
+        configureStorageBackend(builder);
+
+        StorageBackendBuilderValues = builder;
+        return this;
+    }
+
     internal void Reset()
     {
         DefaultValueFactory = null;
+        StorageBackendBuilderValues = null;
+    }
+}
+
+internal interface IStorageBackendBuilderValues
+{
+    public SettingsStorageBackendId BackendId { get; }
+    public Type? BackendType { get; }
```

Comment:
This currently seems unused.
(As far as I can tell, my IDE is broken)

I'm not bothered by it, but figured out I'd point it out.

Reply:
![2024-04-15_13-29-59](https://github.com/Nexus-Mods/NexusMods.App/assets/16577545/36be0913-d160-4dfa-bfa6-6b438c7d28ea)

Reply:
Copy-pasted an if-statement, fixed the usages now.